server/index.js

```diff
@@ -1,21 +1,33 @@
 const express = require('express');
+const bodyParser = require('body-parser');
 const app = express();
 const port = 8080;
 const queryAPI = require('./atelier.js');
+const uploadPhotoToCloudinary = require('./cloudinary.js');
 app.use(express.static('public'));
 app.use(express.json());
+app.use(bodyParser.urlencoded({ extended: false }));
 
 app.all('*', (req, res) => {
-  queryAPI(req.method, req.url, req.body)
-    .then((response) => {
-      let { statusCode } = response.request.res;
-      res.status(statusCode).send(response.data);
-    })
-    .catch((err) => {
-      console.log(err);
-      res.status(500).send(err);
-    });
-
+  if (req.url === '/review/image') {
```

Comment:
I was trying to think if this new server-side process could be encapsulated in our wildcard but I couldn't think of an elegant way to make that happen (we'd probably need some kind of if-then statements in atelier.js if we remove those here). This looks pretty good to me but would be good to get @matthewwrobel and/or @alizehrehman to check this file out just to be sure!

Reply:
Yes, yesterday I was debating between adding additional code directly to index.js (this current version) and modifying @matthewwrobel's queryAPI helper function. I thought it might be better to keep my cloudinary code and Matt's queryAPI separate since they're two separate APIs, but please let me know your thoughts!

Reply:
@tchitrakorn I think that was the right call to not modify the queryAPI function because it's serving a different purpose. Another option to this might be to setup a route to handle requests to /review/image that comes before the wildcard route. All those requests would be responded to before the wildcard route would have a chance to run. But if this is working as expected I think it's fine to merge it as is. Nice job figuring all that out!!

Reply:
example: if you put app.post('/review/image') before the app.all then app.all will never run because the request/response cycle will have ended before it's invoked.

Reply:
@matthewwrobel Everything seems to be working so I'm just gonna go ahead and merge, but your alternative method seems a little cleaner. I will keep this in mind when refactoring later on, thank you!